src/pages/GosuDetail/GosuDetail.js

```diff
@@ -0,0 +1,690 @@
+import React from 'react';
+import styled from 'styled-components';
+import { BsStarFill } from 'react-icons/bs';
+import { BsStarHalf } from 'react-icons/bs';
```

Comment:
```suggestion
import { BsStarFill, BsStarHalf } from 'react-icons/bs';
```
같은 import 경로 끼리는 묶어줄 수 있을 것 같습니다.

Reply:
넵!